helix-term/src/commands.rs

```diff
@@ -3858,32 +3860,46 @@ fn match_brackets(cx: &mut Context) {
 
 //
 
-fn jump_forward(cx: &mut Context) {
+fn jump_impl(cx: &mut Context, copy_cursor: bool, direction: Direction) {
     let count = cx.count();
-    let view = view_mut!(cx.editor);
+    let (view, doc) = current!(cx.editor);
+    let mut this_selection = doc.selection(view.id).clone();
```

Comment:
This clone is not required if `copy_cursor` is true, could it be avoided in some elegant way?